Docs/MoltenVK_Configuration_Parameters.md

```diff
@@ -676,3 +676,14 @@ features that are difficult to support otherwise.
 Unlike `MVK_USE_METAL_PRIVATE_API`, this setting may be overridden at run time.
 
 This option is not available unless MoltenVK were built with `MVK_USE_METAL_PRIVATE_API` set to `1`.
+
+---------------------------------------
+#### MVK_CONFIG_SHADER_DUMP_DIR
+
+##### Type: String
+##### Default: `""`
+
+_(The default value is an empty string)._
+
+If not empty, MoltenVK will dump all SPIRV shaders, compiled MSL shaders, and pipeline shader lists to the given directory.
+The directory will be non-recursively created if it doesn't already exist.
```

Comment:
As an alternate to `mkdir()`, consider using `std::filesystem:: create_directories()`, which will recursively create all the directories in the path.